soc/xtensa/nxp_adsp/imx8ulp/Kconfig.defconfig.series

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2023 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_SERIES_NXP_IMX8ULP
+
+config SOC_SERIES
+	string
+	default "imx8ulp"
+
+config SOC_TOOLCHAIN_NAME
+	string
+	default "nxp_imx8ulp_adsp"
+
+config SOC
+	string
+	default "nxp_imx8ulp"
+
+config SYS_CLOCK_HW_CYCLES_PER_SEC
+	default 666000000 if XTENSA_TIMER
+
```

Comment:
@TangleZ what is the DSP frequency for imx8ulp? Looking at src/platform/imx8ulp/lib/clk.c it should be 528 MHz so please fix SYS_CLOCK_HW_CYCLES_PER_SEC

Reply:
Correct it to 528000000